specs/http-client/index.html

```diff
@@ -233,23 +238,23 @@ <h2>The <dfn>http:send-request</dfn> function</h2>
       <ul>
         <li><code>$request</code> contains the various parameters of the request, for instance the
           HTTP method to use or the HTTP headers. Among other things, it can also contain the other
-          param's values: the URI and the bodies. If they are not set as parameter to the function,
-          their value in $request, if any, is used instead. See the following section for the
-          detailed definition of the http:request element. If the parameter does not follow the
-          grammar defined in this spec, this is an error <a href="#dfn-errHC005">err:HC005</a>.</li>
-        <li><code>$href</code> is the HTTP or HTTPS URI to send the request to. It is an xs:anyURI,
+          parameter’s values: the URI and the body(s). If they are not set as a parameter to the function,
+          their value in <code>$request</code>, if any, is used instead. See the <a href="#request-element" class="sectionRef"></a> for the
+          definition of the <a>http:request</a> element. If the parameter does not follow the
+          grammar defined in this spec, the error <a>err:HC005</a> MUST be raised.</li>
+        <li><code>$href</code> is the HTTP or HTTPS URI to send the request to. It is an <code>xs:anyURI</code>,
           but is declared as a string to be able to pass literal strings (without requiring to
-          explicitly cast it to an xs:anyURI).</li>
+          explicitly cast it to an <code>xs:anyURI</code>).</li>
         <li><code>$bodies</code> is the request body content, for HTTP methods that can contain a
-          body in the request (e.g. POST). This is an error if this param is not the empty sequence
+          body in the request (e.g. POST). It is an error if this parameter is not the empty sequence
           for methods that must be empty (e.g. DELETE). The details of the methods are defined in
-          their respective specs (e.g. [[!rfc2616]] or [[!rfc4918]]). In case of a multipart
+          their respective specifications (e.g. [[!rfc2616]] or [[!rfc4918]]). In case of a multipart
           request, it can be a sequence of several items, each one is the body of the corresponding
           body descriptor in
-          <code>$request</code>. See below for details.</li>
+          <code>$request</code>; See <a>http:multipart</a>.</li>
       </ul>
-      <p>Besides the 3-params signature above, there are 2 other signatures that are convenient
-        shortcuts (corresponding to the full version in which corresponding params have been set
+      <p>Besides the arity 3 signature above, there are two other signatures that are convenient
```

Comment:
"arity 3" > "arity-three" (this follows W3C spec style; see 2nd paragraph of https://www.w3.org/TR/xpath-functions-31/#accessors, e.g. "Each of these functions has an *arity-zero* signature which is equivalent to the *arity-one* form...")